### DISCUSSION TITLE
request for new release with fltk 6a5c326c39e9b2353ca9439ba1e9dab2815e6000

### DISCUSSION THREAD

**Discussion by cstkingkey** [fltk-rs/fltk-rs#1339] [General]
MoAlyousef, can you make a new release with fltk 6a5c326c39e9b2353ca9439ba1e9dab2815e6000. It fixes the deadlock issue I ran into.

**Comment by MoAlyousef** [comment]
Sure, I’ll publish a new version in a few hours.

**Reply by cstkingkey** [reply]
> Sure, I’ll publish a new version in a few hours.

Thanks.
